File: Conceitos de LPs/Tipos de Dados/Registros/Pessoa.kt
//Kotlin
//Não tem registros. Classe é uma espécie de um Registro Aprimorado, pois também Abstrai Dados heterogêneos, mas adiciona Encapsulamento, Herança e Polimorfismo.

data class Pessoa(val nome: String, val idade: Int, val altura: Double)

fun main() {
    val pessoa = Pessoa("Bruna", 29, 1.68)

    // Exibindo valores
    println("Nome: ${pessoa.nome}")
    println("Idade: ${pessoa.idade}")
    println("Altura: ${pessoa.altura} m")
}

File: Conceitos de LPs/Tipos de Dados/Registros/Pessoa.rs
//Rust
//Aqui o registro é chamado de Estrutura (struct)

struct Pessoa {
    nome: String,
    idade: u32,
    altura: f32,
}

fn main() {
    let pessoa = Pessoa {
        nome: String::from("Marcos"),
        idade: 27,
        altura: 1.78,
    };

    // Exibindo valores
    println!("Nome: {}", pessoa.nome);
    println!("Idade: {}", pessoa.idade);
    println!("Altura: {} m", pessoa.altura);
}
